.github/workflows/todos-to-issue.yml

```diff
@@ -0,0 +1,18 @@
+name: Todos to issue
+
+on:
+  push:
+    branches:    
+      - master
+
+jobs:
+  todos-to-issue:
+    runs-on: ubuntu-latest
+    name: Looks for TODO comments in the source code and creates an issue with the found TODOs
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Run todos to issue action
+        uses: nearform/github-action-todos-to-issue@master

```

Comment:
would it be better to use the release instead of master?

`uses: nearform/github-action-todos-to-issue@v1`